dashboard/app/views/devise/sessions/new.html.haml

```diff
@@ -44,7 +45,7 @@
           %button#signup-button.secondary{type: 'button'}= t('nav.user.signup')
 
   .span4
-    .sidebar= render "devise/shared/links"
+    .sidebar= render "devise/shared/links" unless @hide_oauth_sidebar
```

Comment:
But I *should* be allowed to go through oauth. The warning warns me that I probably don't want to. But if it is appropriate, I shouldn't need to reload the page (without the query parameter) to access those links.